components/Conversations/Detail/ConversationDetail/ActiveConversation.tsx

```diff
@@ -37,26 +34,22 @@ const ActiveConversation: React.FC<ActiveConversationProps> = ({ onConversationS
 
   // Reset the startTime when the conversation is saved
   useEffect(() => {
-    const removeConversationSavedListener = Highlight.app.addListener(
-      'onConversationSaved',
-      () => {
-        setStartTime(null)
-      }
-    )
+    const removeConversationSavedListener = Highlight.app.addListener('onConversationSaved', () => {
+      setStartTime(null)
+    })
     return () => removeConversationSavedListener()
   }, [])
 
   const handleSaveConversation = async () => {
     try {
       const savedConversation = await saveCurrentConversation()
       if (savedConversation) {
-        onConversationSaved(savedConversation)
+        handleConversationSelect(savedConversation.id)
```

Comment:
`handleConversationSelect` already exists as a function in `useConversations` and is used when clicking on the saved conversations in the left panel. We can just reuse that here